.takt/facets/instructions/review-pekko-compat.md

```
@@ -1,4 +1,7 @@
 **Pekko互換性**と**fraktor-rs規約準拠**のレビューに集中してください。
+
+## やらないこと (Do Not)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)